src/scss/variables-admin.scss

```diff
@@ -0,0 +1,49 @@
+/* ==========================================================
+ * admin-variables.scss
+ * Variables set for our project
+ *
+ * Author: Yann Gouffon, yann@antistatique.net
+ * Date:   2014-04-30 09:38:36
+ *
+ * Copyright 2014 Federal Chancellery of Switzerland
+ * Licensed under MIT
+ =========================================================== */
```

Comment:
Maybe a little comment that this comes from some GitHub repo would be great (if this style variables were to be changed there for instance)

Reply:
Good idea. I'll add one.